mscore/menus.cpp

```diff
@@ -1603,6 +1604,10 @@ PalettePanel* MuseScore::newTextPalettePanel(bool defaultPalettePanel)
       stxt->setXmlText(QT_TRANSLATE_NOOP("Palette", "System Text"));
       sp->append(stxt, QT_TRANSLATE_NOOP("Palette", "System text"))->setElementTranslated(true);
 
+      MeasureNumber* meaNum = new MeasureNumber(gscore);
+      meaNum->setXmlText(QT_TRANSLATE_NOOP("Palette", "Bar Number"));
+      sp->append(meaNum, QT_TRANSLATE_NOOP("Palette", "Bar/Measure Number"))->setElementTranslated(true);
```

Comment:
Hmmm, "Bar" is more of a British-English term, and since MuseScore's original language is intended to be American-English, better just use "Measure Number" here and leave it to the British translators to translate to "Bar Number" :-)

Reply:
Indeed, please use Measure here and let the translaros do the work for the en_UK translation

Reply:
Indeed, please use "Measure" here and let the translators (me ;-)) do the work for the en_UK translation

Reply:
should I leave the description to still contain both?

Reply:
No, leave that to the translation

Reply:
If you say so. But I can't find any good reason for that.

Reply:
It is a translatable text. Original language (and the one to ge used thoughout the source) is American English. "bar" is British English

Reply:
So if an American clicks the palettes' search box and types bar number, he won't find anything?
Bar number should be commonly understood, no? Isn't (in this case) too much better than not enough?

Reply:
an American won't search bar in MzuseScore

Reply:
haha, ok.